episodes/11-exercises.md

```diff
@@ -114,33 +142,96 @@ and that assesses their ability to do so.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-Well-designed exercises are one of the most valuable resources for an Instructor:
-time spent on this is well invested.
+## Demo of Writing an Exercise 
+
+Well-designed exercises are one of the most valuable resources for an instructor and 
+any time spent on this is well invested.
+
+<!--- will there be an example lesson that the trainer is working on ? -->
+
+To create an exercise in the Carpentries lesson infrastructure, 
+you can use colon-delimited sections called 'fenced divs'.
+In fact there are many types of boxes in the lesson infrastucture, called "special blockquotes", 
+that use fenced divs.
+To start a challenge fenced div the line must contain at least 3 colons, then the `challenge` tag.
+Then the content of the challenge is included on the following lines.
+Finally, you can close the fenced div using another line with least 3 colons.
+
+```
+:::::::::::::::::::::::::::::::::::::: challenge
+
+### Challenge Title
+
+Challenge text, code, and other information goes here
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+```
+
+If you also want to include an expandable solution box for the challenge you can 
+add a solution fenced div within the challenge box.
+The format is the same as for a challenge except the tag is `solution` instead. 
+Note the solutions can all be expanded for more accessible reading using the "Expand All Solutions"
+option at the top of each episode.
+
+
+```
+:::::::::::::::::::::::::::::::::::::: challenge
+
+### Challenge Title
+
+Challenge text, code, and other information goes here
 
-Demonstration of how to write an exercise within the lesson infrastructure,
-including how to add a solution box.
+:::::::::::::: solution
+
+### Solution Title
+
+Solution text, code, and other information
+
+:::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+```
+
+For readability, you may want to try to match have final lines match up in length to
+the opening lines.
```

Comment:
I want to avoid people assuming that the number of colons needs to match up exactly. 

```suggestion
For readability, you may want to try to match have final lines approximately match up 
in length to the opening lines.
```